support-files/mysql-multi.server.sh

```diff
@@ -157,15 +157,15 @@ case "$mode" in
     # root password.
     if test -f "$pid_file"
     then
-      mysqld_pid=`cat $pid_file`
```

Comment:
`mysqld_pid` is renamed here, but in `mysql.server.sh` it is not. We should either change it there as well or leave variables as they are everywhere.